src/main/scala/com/fulcrumgenomics/umi/DuplexConsensusCaller.scala

```diff
@@ -139,6 +139,14 @@ class DuplexConsensusCaller(override val readNamePrefix: String,
         val (_, abR1s, abR2s) = subGroupRecords(ab)
         val (_, baR1s, baR2s) = subGroupRecords(ba)
 
+        // The orientation of AB and BA reads should be:
+        // AB R1: +  AB R2: -
```

Comment:
Is this true?  I ask because I honestly can't recall whether `GroupReadsByUmi` constrains things so that AB are always +/- and BA are always -/+, or just that they're the opposite of each other.